docs/reference/operations-guide/export.md

```diff
@@ -1322,6 +1322,14 @@ Fetches data from a Gaffer result cache. Requires a cache to be [configured](../
 !!! example
     See examples for ExportToGafferResultCache above for usage.
 
+## GetExports
+
+Fetches multiple export results and returns them in a map. [Javadoc](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/operation/impl/export/GetExports.html)
+
+!!! example
+    See examples for ExportToGafferResultCache above for usage.
```

Comment:
It's only the final example that contains this operation. So it would be better to give that one by name.